app/src/main/shared/styles/buttons.scss

```diff
@@ -1,28 +1,18 @@
+@import "bootstrap";
+
 button, .button {
-    display: inline-block;
-    border: none;
-    background-color: rgb(0, 173, 239);
-    color: white;
-    font-family: inherit;
-    font-size: 12pt;
+    @include button-variant(rgb(0, 173, 239), rgb(200, 200, 200));
+    @include button-size(5px, 20px, 12px, 24px,  0);
     font-weight: bold;
-    padding: 6px 20px 5px 20px;
-    vertical-align: top;
-    outline: rgb(200, 200, 200) outset 1px;
+    @extend .d-inline-block;
+    @extend .align-top;
 }
 
 button[type=submit], .submit{
-    color: #fff;
-    background-color: #5cb85c;
-    outline-color: #4cae4c;
-}
-
-button:hover, .button:hover {
-    outline-width: 2px;
-}
-
-button:active {
-    outline-style: inset;
+    @include button-variant(#5cb85c, #4cae4c);
+    @include button-size(5px, 20px, 12pt, 24px,  0);
+    outline: #4cae4c outset 1px;
+    font-weight: bold;
```

Comment:
Ditch this outline property and you don't need font-weight bold as `.submit` should always be applied to a button. So really should be `button.submit, .button.submit`